tests/test_indexer.py

```diff
@@ -185,6 +253,32 @@ def load_test_data_bundle(self, bundle: S3TestBundle):
         self.upload_files_and_create_bundle(bundle)
         return f"bundles/{bundle.uuid}.{bundle.version}"
 
+    def subscribe_for_notification(self, query, callback_url):
+        url = str(UrlBuilder()
+                  .set(path="/v1/subscriptions")
+                  .add_query("replica", "aws"))
+        resp_obj = self.assertPutResponse(
+            url,
+            requests.codes.created,
+            json_request_body=dict(
+                query=query,
+                callback_url=callback_url),
+            headers=self._get_auth_header()
+        )
+        uuid_ = resp_obj.json['uuid']
+        return uuid_
+
+    def _get_auth_header(self, real_header=True):
```

Comment:
IMHO "real" here is loose nomenclature. I'd make it more explicit with `get_auth_header(token=None)`. If token is None, then a real token will be generated. That way I can pass a fake token easily and it will be more obvious to the reader what I'm doing.